components/country/overview-charts.js

```diff
@@ -72,42 +72,154 @@ class TestsByGroup extends React.PureComponent {
     }))
   }
 
+
   render() {
-    let testCoverageMaxima, networkCoverageMaxima
+    const { testCoverage, networkCoverage } = this.props
+    
+    // Check if there is enough data to plot the charts
+    const testCoverageCount = testCoverage.reduce((count, item) => count + item.count, 0)
+    const networkCoverageCount = networkCoverage.reduce((count, item) => count + item.count, 0)
```

Comment:
This is maybe not so efficient and we should maybe consider adding an API endpoint to serve this need.

I documented this here: https://github.com/ooni/api/issues/99